### DISCUSSION TITLE
Move the auto login button into the navigation menu

### DISCUSSION THREAD

**Discussion by szaimen** [nextcloud/all-in-one#367] [Ideas] [answered]
https://github.com/nextcloud/activity/blob/master/appinfo/info.xml#L66
https://docs.nextcloud.com/server/latest/developer_manual/app_development/init.html
https://docs.nextcloud.com/server/latest/developer_manual/basics/routing.html?highlight=rout#using-the-urlgenerator

increase the php app version then, too

when done, also change the documentation: https://github.com/nextcloud/all-in-one#how-to-easily-log-in-to-the-aio-interface

(this change was suggested by jan)
Question is if a settings page isn't still better because you can delegate the administration then...

**Comment by ghost** [comment]
Why should the autologin button be moved? I find the place quite good where it is.

**Comment by ghost** [comment, marked as answer]
The button does not have such a high priority that it should be accessible from everywhere. Especially because the time-controlled elements in the AIO interface automate a lot of things, which means that the administrator has to switch to the AIO interface even less often.

**Reply by szaimen** [reply]
This is actually a very good point! Thanks for this!
Also with the current position, admin delegation is possible which is not when moving the button into the navigation menu, afaik.
